File: toMD5withSalt.py
#!/usr/bin/env python3

import sys
from hashlib import md5

salt = b'jirlaogbfj'

# input
text = []
for l in sys.stdin:
    text.append(l.replace('\n','').encode('utf-8'))
num = len(text)
for i in range(num):
    salt_text = text[i] + salt
    hash_text = md5(salt_text).hexdigest()
    print(hash_text)

File: Encryption/toMD5withSalt.php
<?php

$salt = 'jirlaogbfj';
// print(mb_detect_encoding($salt));
// input
while ($line = fgets(STDIN)) {
   $text[] = trim($line);
}

$num = count($text);
for ($i = 0; $i < $num; $i++) {
    //print "$text[$i]¥n";
    $salt_text = $text[$i].$salt;
    $hash_text = hash('md5', $salt_text);
    echo $hash_text."\n";
}
?>